kombu/transport/SQS.py

```diff
@@ -160,15 +152,23 @@ def _reset_cycle(self):
 
     def entity_name(self, name, table=CHARS_REPLACE_TABLE):
         """Format AMQP queue name into a legal SQS queue name."""
-        return text_t(safe_str(name)).translate(table)
```

Comment:
So, it is basically an enhancement, adding SQS FIFO support :) . I believe it belongs to a separate Pull Request, but I will not insist any further.